<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>241</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Список задач</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Описание задачи</string>
    </property>
    <property name="buddy">
     <cstring>textAbout</cstring>
    </property>
   </widget>
   <widget class="QTextEdit" name="textAbout">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>481</width>
      <height>301</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkCompl">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Выполнено</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkDue">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Просрочено</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="butAdd">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить задачу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butChange">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>350</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Редактировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butDel">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>380</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listTask">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>251</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
